o4/o4_pyforce.py

```diff
@@ -30,7 +30,11 @@ def __init__(self, *args):
         self.stderr = NamedTemporaryFile()
         if os.environ.get('DEBUG', ''):
             print(f'## p4', *self.args, file=sys.stderr)
-        self.pope = Popen(['p4', '-vnet.maxwait=60', '-G'] + self.args,
+        try:
+            timeout = abs(int(os.environ['O4_P4_TIMEOUT']))
+        except:
```

Comment:
A nice example of where catch all makes more sense than enumerating keyerror, valueerror etc.